src/cli_support.ts

```diff
@@ -10,7 +10,7 @@ import * as path from 'path';
 
 // Postprocess generated JS.
 export function pathToModuleName(context: string, fileName: string): string {
-  fileName = fileName.replace(/\.js$/, '');
+  fileName = fileName.replace(/\.[^/.]+$/, '');
```

Comment:
why was this change needed?

Reply:
I started asking for the module name given the path to a .ts file, not just the .js files that were handled before.